avm/res/kubernetes-configuration/flux-configuration/main.bicep

```diff
@@ -24,7 +24,7 @@ param configurationProtectedSettings object?
 @description('Conditional. Parameters to reconcile to the GitRepository source kind type. Required if `sourceKind` is `GitRepository`.')
 param gitRepository object?
 
-@description('Optional. Array of kustomizations used to reconcile the artifact pulled by the source type on the cluster.')
+@description('Required. Array of kustomizations used to reconcile the artifact pulled by the source type on the cluster.')
```

Comment:
This should require a minor version bump (breaking change until v1.0.0 is out)

Reply:
True true